mirdata/tinysol.py

```diff
@@ -0,0 +1,301 @@
+# -*- coding: utf-8 -*-
+"""TinySOL Dataset Loader.
+
+TinySOL is a dataset of 2478 samples, each containing a single musical note from one of 14 different instruments:
+
+    Bass Tuba
+    French Horn
+    Trombone
+    Trumpet in C
+    Accordion
+    Contrabass
+    Violin
+    Viola
+    Violoncello
+    Bassoon
+    Clarinet in B-flat
+    Flute
+    Oboe
+    Alto Saxophone
+
+
+These sounds were originally recorded at Ircam in Paris (France) between 1996
+and 1999, as part of a larger project named Studio On Line (SOL). Although SOL
+contains many combinations of mutes and extended playing techniques, TinySOL
+purely consists of sounds played in the so-called "ordinary" style, and in
+absence of mute.
+
+TinySOL can be used for education and research purposes. In particular, it can
+be employed as a dataset for training and/or evaluating music information
+retrieval (MIR) systems, for tasks such as instrument recognition or
+fundamental frequency estimation. For this purpose, we provide an official 5-
+fold split of TinySOL. This split has been carefully balanced in terms of
+instrumentation, pitch range, and dynamics. For the sake of research
+reproducibility, we encourage users of TinySOL to adopt this split and report
```

Comment:
What about including the split as metadata per track?

Reply:
This could be done by adding a second metadata file containing just the splits, and loading it alongside the existing metadata

Reply:
@rabitt the split is a column in `TinySOL_metadata.csv`, indeed.
did you understand otherwise? if so, i need to clarify the docstring

Reply:
update: i added another sentence in the docstring to make clear that the Fold ID is part of the metadata.